--- annobin-13.25,orig/gcc-plugin/annobin.cc	2026-07-22 09:39:03.054591918 +0100
+++ annobin-13.25/gcc-plugin/annobin.cc	2026-07-22 09:39:38.086821040 +0100
@@ -2192,7 +2192,7 @@ annobin_emit_start_sym_and_version_note
 	     Ensure that we do not have empty special text sections so that the
 	     annobin start symbols are never beyond the end of the sections.  */
 #ifndef ANNOBIN_NOP_INSN
-#define ANNOBIN_NOP_INSN ".nop"
+#define ANNOBIN_NOP_INSN "nop"
 #endif
 	  if (* suffix && enable_ppc64_nops)
 	    annobin_emit_asm (ANNOBIN_NOP_INSN,
